pytorch_lightning/trainer/lr_finder.py

```diff
@@ -53,6 +53,7 @@ def _run_lr_finder_internally(self, model: LightningModule):
     def lr_find(self,
                 model: LightningModule,
                 train_dataloader: Optional[DataLoader] = None,
+                val_dataloaders: Optional[DataLoader] = None,
```

Comment:
do we need to add also `test_dataloaders`

Reply:
no, the bug happens due to this check:
https://github.com/PyTorchLightning/pytorch-lightning/blob/142bc0230e228cd2e851481e5a07069e7d198655/pytorch_lightning/trainer/trainer.py#L1009-L1012
which is only present for the validation dataloader.